app/src/main/java/com/android/streetworkapp/ui/progress/Progression.kt

```diff
@@ -76,12 +79,17 @@ fun ProgressScreen(
     navigationActions: NavigationActions, // Note: not used yet
     userViewModel: UserViewModel,
     progressionViewModel: ProgressionViewModel,
+    workoutViewModel: WorkoutViewModel =
+        WorkoutViewModel(WorkoutRepositoryFirestore(FirebaseFirestore.getInstance())),
```

Comment:
I'm not sure it's a good idea to leave the possibility of implicitly declaring the viewmodel here, it should be done in the MainActivity to avoid multiple instances. Do you agree with me?

Reply:
You are right, I removed it and I updated the tests.